components/Schedule.tsx

```diff
@@ -75,169 +75,155 @@ const Schedule = ({
         ) ?? ({ popularity: ClassPopularity.Unknown } as ActivityPopularity);
 
     return (
-        <Stack direction={"row"}>
-            {schedule.days.map((day) => (
-                <Box key={day.date} width={200} pr={2}>
-                    <Box py={2} width={200}>
-                        <Typography variant="h6" component="div">
-                            {day.dayName}
-                        </Typography>
-                        <Typography
-                            variant="h6"
-                            component="div"
-                            style={{
-                                color: theme.palette.grey[600],
-                                fontSize: 15,
-                            }}
-                        >
-                            {day.date}
-                        </Typography>
+        <Stack>
+            <Stack direction={"row"} margin={"auto"} spacing={2}>
+                {schedule.days.map((day) => (
+                    <Box key={day.date} width={200}>
```

Comment:
These lines are the only ones that have really been modified. Prettier did a lot of heavy lifting indenting due to the extra `<Stack>...</Stack>` wrapper around the content